src/FixtureBuilder/ExpressionLanguage/Lexer/ReferenceEscaperLexer.php

```diff
@@ -44,7 +44,7 @@ public function __construct(LexerInterface $decoratedLexer)
      */
     public function lex(string $value): array
     {
-        $escapedValue = preg_replace('/((\s|^)[a-zA-Z0-9_.+-]+?[A-Za-z0-9])@/', '$1\\@', $value);
+        $escapedValue = preg_replace('/((\s|^)[^@]?(\p{L}|\p{N}|[+-_.])+?(\p{L}|\p{N}))@/', '$1\\@', $value);
```

Comment:
If the regex was written without the optional `[^@]` it would work fine but only emails with at least 2 characters in the name would be matched.